tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot

```diff
@@ -33,69 +33,74 @@ Launch Container
 
 *** Test Cases ***
 Upgrade VCH with containers
-    Log  Disabled until PR \#2554 is merged  WARN
-    # ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} network create bar
-    # Should Be Equal As Integers  ${rc}  0
-    # Comment  Launch first container on bridge network
-    # ${id1}  ${ip1}=  Launch Container  vch-restart-test1  bridge
-    # ${id2}  ${ip2}=  Launch Container  vch-restart-test2  bridge
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} network create bar
+    Should Be Equal As Integers  ${rc}  0
+    Comment  Launch first container on bridge network
```

Comment:
Does this comment need to be updated? We are launching not one but two containers below.